index.src.html

```diff
@@ -290,6 +290,16 @@
     </ul>
   </section>
 
+  <section algorithm="'create a credential' internal method">
+    <dfn for="Credential" method export>\[[Create]](data)</dfn> is called with a
+    {{CredentialCreationData}}, and returns either a {{Credential}}, if one can be created, or
```

Comment:
`[[Create]]` should be able to return a `Promise<whatever>` in addition to the synchronous `whatever` ... or maybe it should always return a Promise.